app/assets/stylesheets/custom.scss

```
@@ -28,7 +28,9 @@ a {
 		text-decoration: none;
 	}
 }
-
+.text_dark{
```

```suggestion
.text_dark {
```
sai indent nhé
tab 2 space thôi

---

vang a hichic